data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+		personality heroic vindictive staying uninterested
+		system "Eogho"
+		ship "Felsic" "Tschelgi Dueich"
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Vi (Neutral)"
+		personality heroic vindictive staying uninterested swarming
+		system "Eogho"
+		fleet
+			names "gegno vi"
+			variant
+				"Hornfel" 2
+				"Eclogite"
+				"Slate"
+	npc
+		on provoke
+			set "interrupted arena"
+		government "Gegno Scin (Neutral)"
+		personality heroic vindictive staying uninterested swarming
+		system "Eogho"
+		fleet
+			names "gegno scin"
+			variant
+				"Coesite" 2
+				"Tridymite"
+				"Mica" 2
+
+
+mission "Brief Bystander"
+	invisible
+	landing
+	destination "Esstch"
+	to offer
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on complete
+		"gegno: encounters" ++
+		conversation
+			`The presence of scattered recording instruments on the surface of this planet contradicts what little interest the Gegno appear to be giving it nowadays, despite its location within the Gegno's home system of Heutesl. Still, since this planet isn't strictly owned by the Gegno, it makes for a convenient place to stop, and they seem to neither notice nor care that you have landed here.`
+			`	However, as you fly around, you notice a small Gegno shuttle parked on the ground. The tan coloration of its hull suggests that it's a civilian craft, but what it's doing here is unknown.`
+			choice
+				`	Investigate.`
+					goto investigate
+				`	Leave them be.`
+					decline
+
+			label investigate
+			action
+				set "gegno: approached bystander"
+			`	You steer your ship towards the shuttle, and once you approach it, you are able to see a Gegno on the ground fiddling with one of the nearby devices. Though, as you fly closer, they stop their work and return to their shuttle, taking off shortly after. It seems like they weren't interested in any potential interaction with you.`
+			`	The satellite they were tinkering with doesn't strike you as anything special, being just another antique device amongst the others.`
+				decline
+
+
+mission "Scin on Cyife"
+	invisible
+	landing
+	source "Cyife"
+	to offer
+		has "First Contact: Gegno Scin: offered"
+		not "Gegno Genocide Defense: offered"
+	on offer
+		"gegno: encounters" ++
+		conversation
+			`Nothing is of much interest on a cold, unwelcoming moon in the middle of nowhere. However, there's something about the surface of Cyife that is a bit peculiar. Some of the light refracted through the surface seemingly dissipates or reflects back in strange patterns. Although it appears to be ice, more detailed inspection reveals that small patches of the terrain seem to instead be made of some other type of crystalline material. In the distance is something that truly stands out on this world - a large egg-shaped structure with a green hue to it.`
+
+			branch "saw epidote"
+				has "scin: approached epidote"
+
+			branch "boarded epidote"
+				has "scin: entered epidote"
```

Comment:
Playtesting revealed a bug here. If you entered the epidote, you would still get shunted to the `saw epidote` label because it comes first in the code. Suggesting you reverse the order, or the player has no way of reaching `boarded epidote`.
```suggestion
			branch "boarded epidote"
				has "scin: entered epidote"

			branch "saw epidote"
				has "scin: approached epidote"
```